### DISCUSSION TITLE
Encourager des cantines qui sont avancée niveau EGAlim d'écrire des articles pour notre blog

### DISCUSSION THREAD

**Discussion by hfroot** [betagouv/ma-cantine#996] [Idées]
Selon les réponses de diagnostics, peut-être on peut afficher un formulaire de contact en encourageant la cantine de partager ses connaissances sur notre blog.

(deuxième pas : relier un article avec une cantine, par exemple ajouter un lien sur la page cantine publiée si elle a partagé des conseils sur le blog)

**Comment by alemangui** [comment]
Ça m'intéresserait de savoir ce qu'on veut que le blog devienne (@JenniferStephan et @valeriemerle). My two cents :

Dans un premier temps j'ai cru comprendre que le blog était là pour montrer le parcours de certaines cantines - mais depuis qu'on a la publication avec des champs libres que les gestionnaires peuvent remplir ce besoin est en partie comblé. Si c'est ça, je ne crois pas que ça soit utile de solliciter davantage l'utilisateur (qui a déjà bcp de choses à remplir, publier, télédéclarer, etc) car ça augmenterait ça charge mentale pour pas grand gain. 

D'un autre coté le blog peut être vu comme une plateforme pour spécifier des éléments de la loi EGAlim et avoir un SEO plus performant. Ça ferait une entrée organique vers la plateforme.

**Reply by gauthierandre** [reply]
Je me permets d'ajouter quelques éléments.

Je te rejoins @alemangui sur le second point, l'espace blog est hyper intéressant pour développer le SEO et peut être une porte d'entrée via ma cantine (même si google Search console démontre clairement que ce n'est pas un point d'entrée important). Dans un second temps, le blog a pour vocation de permettre une forme d'animation du réseau de la restau'co que la seule fonctionnalité de publication ne permet pas. Car, les articles — via les témoignages ou les articles issus de l'expérience des utilisateurs, ou encore les éléments juridiques — permettent de partager voire de centraliser des ressources, des pratiques sur une thématique aux personnes intéressées.

Cela dit, j'ai peur aussi que ça fasse beaucoup d'inviter à saisir des informations. Surtout si on n'est pas très directif sur ce qu'on attend de leur part en termes de contenu. Il faut pas mal d'informations pour un article. Mais pour rejoindre @hfroot, on peut, suivant le diagnostic, imaginer une fenêtre qui s'ouvre pour inviter à nous contacter afin de partager leurs pratiques.

**Reply by hfroot** [reply]
@alemangui je suis d'acc que cette fonctionnalité doit être secondaire d'autres actions qui existe déjà, et très, très facultatif. Je pense qu'on peut adresser ce concerne avec du design attentif. Mais je comprends ton point de gain, est ça vaut le coup de challenger si cette fonctionnalité réponds à un besoin usager avec l'équipe.

Pour rebondir sur de ton second paragraphe @gauthierandre j'imagine dans un premier temps une fonctionnalité comme ça pourrait être un simple mis en contacte avec notre équipe et pas une invitation d'écrire n'importe quoi sans édition de notre côté.

Ou oui, André en lien avec ton idée, peut-être on peut commencer avec un analyse metabase pour trouver des cantines et les solliciter si on veut.
